### DISCUSSION TITLE
Force HDMI on HD300

### DISCUSSION THREAD

**Discussion by craigap1979** [OpenSageTV/Community#117] [SageTV Extenders (HD300,HD200)]
I recently connected an HD300 to a TCL RokuTV and the TV doesn't detect a signal. This has happened to me in the past but I was able to use the component breakout cable to get things working. This TV does not have component inputs, nor can I find my breakout cables. I was hoping I could manually activate the setting within the client files, but I can't find a "Force HDMI" setting anywhere. Not sure where to look. I can't get into Standalone mode without being able to see the display.

**Comment by TGsell1** [comment]
I’ve had similar strange behavior with HD300s.  In fact I have plugged the hdmi output into a computer monitor to restore the signal …. No clue why this works but we have a good number of users on this ‘forum’ who are keeping them alive so it always pays to post.  Sent from my iPhoneOn Mar 6, 2024, at 10:00 AM, craigap1979 ***@***.***> wrote:﻿
It did not work. However, I brought up another TV and plugged in the HDMI-still didn't work; left to try to find the component adapter-can't find it; came back up and found that SageTV was being displayed on the TV. I unplugged the cable and put it into the TV I want it connected to and it worked. I restarted SageTV and it didn't work again. Plugged it back into the other TV-didn't work. I left for 20 min, came back, and the TV was in Standby. Turned it on and it worked again. I plugged it into the other TV again-still working. Turned off the TV then back on-still worked. I haven't tried it since, but I'm wondering if there is some kind of signal that SageTV needs from the TV that it wasn't getting because I already had the TV on, and when It goes off then back on again maybe it gets that signal.
I also took a moment to go into the settings to Force HDMI and found it was already set from when I used it last.

**Comment by Photon0000** [comment]
Just curious if you have looked at the new TVs HDMI settings? The HDMI standards have evolved and they are trying to make HDMI eliminate the need for users to do things like turn connected devices on and off and select inputs. One receiver I tried was so AI smart it knew better than I what I wanted to do and insisted on switching the input back to the BD player after I had manually selected a different input. I could only get what I want if I turned off the power to the BD player. I'm not into fighting the machines. Not how I like to spend my time.

I like technology to do what I tell it to do and nothing else. That receiver was returned for several reasons including this. Other receivers and streaming devices had to have their HDMI-CEC settings disabled as well. HDMI-CEC is not a good or useful feature in my experience. It may get things right some times but it is often wrong. Having it enabled is just a battle against the machine. https://www.nakamichi-usa.com/what-is-hdmi-cec

Also sometimes but not often a different HDMI cable will resolve a HDMI Issue or the problem is just a poor HDMI connector in the device and wiggling the cable will get the image back. Spending lots of money for a HDMI cable won't solve any HDMI problem. You can find well priced cables that support the latest 8K speeds. If you don't need those speeds for the device you are connecting you don't need that level of cable. Sage works works just fine with cables that support HDMI 1.3 which came out in 2006.
